aws-transfer-workflow/aws-transfer-workflow.json

```diff
@@ -357,13 +357,7 @@
   "primaryIdentifier": [
     "/properties/WorkflowId"
   ],
-  "tagging": {
-    "taggable": true,
-    "tagOnCreate": true,
-    "tagUpdatable": true,
-    "cloudFormationSystemTags": true,
-    "tagProperty": "/properties/Tags"
-  },
+  "taggable": true,
```

Comment:
did cfn updated there tagging flags again?

Reply:
Looks like it was just this originally. We added tag metadata to all schemas, but ended up reverting the change to the Workflow schema. The comment on the revert intended to eventually add the metadata back.

It seems that when we did the revert, we did not do it in this GitHub repo as well. That's why the two schemas are out of sync. I can update this PR to keep the tag metadata since we ultimately want it.

Reply:
that will be great, thank you!

Reply:
@vicwuqinyi Spoke with Alex, and the work on generating schemas from Smithy models will add this back anyway. That will be done in the near future, so I think we can just merge this PR as is?

Reply:
sure. sounds good with me